web/src/main/java/org/springframework/security/web/server/util/matcher/AuthenticationConverterServerWebExchangeMatcher.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.server.util.matcher;
```

Comment:
Placing the class here will cause a package tangle since `org.springframework.security.web.server.authentication` already refers to `org.springframework.security.web.server.util.matcher` in `AuthenticationWebFilter`.

I'd recommend that this got into `org.springframework.security.web.server.authentication` instead.

Reply:
@jzheaux thanks, fixed.